src/menu_put_time_bonus.c

```diff
@@ -0,0 +1,34 @@
+/* ************************************************************************** */
+/*                                                                            */
+/*                                                        :::      ::::::::   */
+/*   menu_put_time_bonus.c                              :+:      :+:    :+:   */
+/*                                                    +:+ +:+         +:+     */
+/*   By: gwolf <gwolf@student.42vienna.com>         +#+  +:+       +#+        */
+/*                                                +#+#+#+#+#+   +#+           */
+/*   Created: 2024/01/08 10:46:17 by gwolf             #+#    #+#             */
+/*   Updated: 2024/01/13 10:57:32 by gwolf            ###   ########.fr       */
+/*                                                                            */
+/* ************************************************************************** */
+
+#include "menu_put.h"
+
+t_vec2i	ft_put_time(t_putinfo put, uint64_t last_render_time, bool show_fps)
+{
+	const float	fps = 1000.0 / (last_render_time);
```

Comment:
How come we divide 1k by the time?

Reply:
We want to get the frames per second.
Last_render_time is in millisecond. One second has 1000 ms.
If we divide a second by the time it takes for one frame we get fps